.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+## Types of changes
+<!--
+  What types of changes does your code introduce? Put an `x` in all the boxes
+  that apply:
+-->
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality
+      to change)
+
+## Checklist:
+<!--
+  Go over all the following points, and put an `x` in all the boxes that apply.
+-->
+- [ ] My code follows the code style of this project.
+- [ ] My change requires a change to the documentation.
+  - [ ] I have updated the documentation accordingly. (only if applicable)
+- [ ] I have added tests to cover my changes.
+- [ ] All new and existing tests passed.
+
```

Comment:
I'd suggest adding additional sections like "Overview of changes", "Outstanding questions", "related issues" etc.